spec/models/account/user_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  context 'validation tests' do
+    let!(:user) { build(:user) }
+
+    it "ensures required fields are present" do
+      expect(user.valid?).to eq(true)
```

Comment:
there're better options to validation for true value
- `expect(user).to be_valid` (read more about `be_` matchers)
- `expect(user.valid?).to be_truthy # be_faley` to check for `true/false`-y like values